packages/frontend/src/components/project/Milestones.tsx

```diff
@@ -0,0 +1,51 @@
+import { Milestone } from '@l2beat/config'
+import cx from 'classnames'
+import React from 'react'
+
+import { MilestoneIcon } from '../icons/symbols/MilestoneIcon'
+import { OutLink } from '../OutLink'
+
+export interface MilestonesProps {
+  milestones: Milestone[]
+  className?: string
+}
+
+//TODO: light mode
+//TODO: date format
+//TODO: link arrow
+export function Milestones({ milestones, className }: MilestonesProps) {
+  return (
+    <div className={cx('relative', className)}>
+      <div>
+        <span className="text-2xl font-bold">Project Milestones</span>
+      </div>
+      <div>
+        <div className="absolute left-[15.4px] h-[100%] mt-2">
+          <div className="h-[70%] w-[1px] bg-green-500 " />
+          <div className="h-[30%] w-[1px] bg-gradient-to-b from-green-500" />
+        </div>
```

Comment:
Good job with placing svgs on top of this line. With the vertical line I have few ideas thou. You don't need to use absolute here, maybe 'flex' would be cleaner here?
```suggestion
      <div className='flex'>
        <div className="h-[100%] mt-2">
          <div className="h-[70%] w-[1px] bg-green-500 " />
          <div className="h-[30%] w-[1px] bg-gradient-to-b from-green-500" />
        </div>
```

Reply:
@antooni What about this one? I'm not a fan of this 15.4px here